src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -9388,172 +9413,6 @@ def quantiles_along_axis0(
             )
         )
 
-    def _quantiles_single_col(
```

Comment:
I think you may have messed up the git history here unless you intended to remove this?

Reply:
yep, I am fixing it now